editor/src/clj/editor/outline_view.clj

```diff
@@ -639,7 +639,7 @@
                          (ui/on-click! (partial toggle-visibility! node-outline-key-path)))
                        (proxy-super setGraphic pane)
                        (when-let [[r g b a] color]
-                         (proxy-super setStyle (format "-fx-text-fill: rgba(%d, %d, %d %d);" (int (* 255 r)) (int (* 255 g)) (int (* 255 b)) (int (* 255 a)))))
+                         (.setStyle text-label (format "-fx-text-fill: rgba(%d, %d, %d %d);" (int (* 255 r)) (int (* 255 g)) (int (* 255 b)) (int (* 255 a)))))
```

Comment:
Now that we have additional controls such as the visibility toggle surrounding each tree item in the Outline View, this style should no longer be applied to the root-level graphic produced for the `TreeItem`. It should be applied to the `Label` itself.